### PR TITLE
Tests/add event listeners

### PR DESCRIPTION
**Pull request by TxHawks**
his pull request adds tests to `addEventListeners()`, as well as fixes a bug found in the code.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -8,9 +8,15 @@ module.exports = {
...
   "env": {
+    "browser": true,
     "es6": true,
+    "mocha": true,
     "node": true,
-    "browser": true
+  },
+
+  "globals": {
+    "expect": true,
```
Comment (nadavsinai) [inline .eslintrc.js]: this is not needed, I inject the  globals via webpack eslint loader - see test-web and test-node configs
setting this here will mean that eslint will think you're in mocha all the time - also in dev mode...

File: src/lib/addEventListeners.spec.js [hunk 2]
```diff
@@ -1,19 +1,221 @@
...
+
+      // Cleanup
+      wrapper.remove();
+    });
   });
-  it('adds next/prev event listeners to buttons found via attributes');
 
-  it('adds onKeyDown listener to the wrapper for keyCode 27 if element is visible');
+  describe('Keydown events on the wrapper', () => {
+    it('adds keydown listener to the wrapper', () => {
```
Comment (nadavsinai) [inline src/lib/addEventListeners.spec.js]: well, you're right this is brittle. let's remove this test
we can just check behaviour, as I said, strictly speaking this is not completely a unit test, but it does not cross to internal implementation which is better than the current test, 
just perhaps rephrase the test below  to be:
it('makes a visible dialog hidden using hide() when pressing 'esc')

File: src/lib/addEventListeners.spec.js [hunk 3]
```diff
@@ -1,19 +1,221 @@
...
+      addEventListeners(dialogState);
+      simulant.fire(wrapper, 'keydown', { keyCode: 9 });
+
+      expect(keepFocusSpy).to.have.been.called();
+
+      // Cleanup
+      wrapper.remove();
+    });
+
+    it('Does not execute keepFocus on keydown when keyCode is 9 & wrapper is *NOT* visible', () => {
```
Comment (nadavsinai) [inline src/lib/addEventListeners.spec.js]: for the sake of DRYness we can make these assertions within the test above using the same setup and cleanup or abstract these and only change the keyCode and assertion

**Comment by nadavsinai** [comment]
lets just revert the eslint globals then OK to merge

File: .eslintrc.js (hunk 1 above)
Comment (TxHawks) [reply .eslintrc.js]: If I remove this I get `warning  'sinon' is not defined` and `warning  'expect' is not defined` when running `npm run lint`
```

**Comment by TxHawks** [comment]
All comments addressed, except the one about changes in `.eslintrc.js` (See comment [here](https://github.com/nadavsinai/htz-dialog/pull/7/files#r97880639))

**Pull request merged**